src/static/riot/display_statistics/show_stats.tag

```diff
@@ -0,0 +1,165 @@
+<show-stats>
+    <button id="stats-btn" onclick="{ stats_button_clicked }"
+            class="ui black big launch left attached {fixed: show_stats} button">
+        <i class="icon {minus: !show_stats, 'chart bar': show_stats}"></i>
+        <span class="btn-text">Stats</span>
+    </button>
+    <div id="stat-card" show="{ !show_stats }" class="ui fixed card">
+        <div class="header-bg content">
+            <div class="header">By the numbers...</div>
+        </div>
+        <div class="stat-content content">
+            <h4 class="ui sub blue header">{producer_name} brings together</h4>
+            <div class="ui three column grid">
+                <div class="column" each="{ stat in general_stats }" no-reorder>
+                    <div class="ui seven tiny statistics">
+                        <div class="statistic">
+                            <div class="value">
+                                { stat.count }
+                            </div>
+                            <div class="label">
+                                { stat.label }
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <script>
+        var self = this
+        self.show_stats = false
+        self.general_stats = {}
+
+        self.on("mount", function () {
+            $(".tooltip", self.root).popup()
+            self.get_general_stats()
+        })
+
+        self.get_general_stats = function () {
+            var endpoint = CODALAB.api.by_the_numbers()
+            endpoint
```

Comment:
just call the api directly w/out the variable as a middleman

Reply:
Fixed in latest push